src/common/FrameManager.ts

```diff
@@ -213,6 +219,20 @@ export class FrameManager extends EventEmitter {
     return watcher.navigationResponse();
   }
 
+  private async _onFrameMoved(event: Protocol.Target.AttachedToTargetEvent) {
+    if (event.targetInfo.type !== 'iframe') {
+      return;
+    }
+
+    // TODO(sadym): Remove debug message oncle the proper OOPIF is implemented:
+    // https://github.com/puppeteer/puppeteer/issues/2548
+    debugFrames(
```

Comment:
Since we only need `debugFrames` once, let’s move its definition here.

Reply:
done